modules/service/src/main/scala/lucuma/odb/api/service/Connection.scala

```diff
@@ -93,11 +98,20 @@ object Connection {
     def stop(id: String): (ConnectionState[F], F[Unit])
 
     /**
-     * Terminates the connection, closing all ongoing subscriptions as well.
+     * Stops all the subscriptions.
      *
      * @return state transition and action to execute
      */
-    def terminate: (ConnectionState[F], F[Unit])
+    def stopAll: (ConnectionState[F], F[Unit])
```

Comment:
What used to terminate the connection now just stops all the active subscriptions.